server/node/README.md

```diff
@@ -7,3 +7,69 @@ Start the server by running:
 ```bash
 npm start
 ```
+
+---
+
+### `GET /paypal-api/auth/browser-safe-client-token`
+
+**Description:**  
+Generates and returns a browser-safe client token for initializing the PayPal JavaScript SDK on the client side.
+
+**Responses:**
+
+- `200 OK`  
+  Returns a JSON object with the client token.
+
+  ```json
+  {
+    "client_token": "abc123..."
+  }
+  ```
+
+
+---
+
+### `POST /paypal-api/checkout/orders/create`
+
+**Description:**  
+Creates a new PayPal order with sample data. This simulates the beginning of a checkout session.
+
+**Request Body:**
+No body required.
+
+**Responses:**
+
+- `201 Created`  
+  Returns the new order's details.
+
+  ```json
+  {
+    "id": "ORDER-ID",
+    "status": "CREATED"
+  }
+  ```
+
+
+---
+
+### `POST /paypal-api/checkout/orders/{orderId}/capture`
+
+**Description:**  
+Captures an existing PayPal order and finalizes the payment.
+
+**Path Parameters:**
+
+- `orderId` (string): The ID of the order to be captured.
+
+**Responses:**
+
+- `200 OK`  
+  Returns capture confirmation.
+
+  ```json
+  {
+    "id": "ORDER-ID",
+    "status": "COMPLETED"
```

Comment:
I think we should mention that the capture response contains additional order information. We could technically link out to here if you'd think that would be helpful: https://developer.paypal.com/docs/api/orders/v2/#orders_capture
```suggestion
    "status": "COMPLETED",
    // additional order information
```